examples/kafka/start.sh

```diff
@@ -0,0 +1,51 @@
+#startup zookeeper
+cd /root/zookeeper-3.4.12/conf
+sed -i '14d' zoo.cfg
+sed -i "13a clientPort=$PAI_CONTAINER_HOST_zookeeper_PORT_LIST" zoo.cfg
+cd /root/zookeeper-3.4.12/bin
+./zkServer.sh start
+sleep 5s
+
+#startup kafka
+cd /root/kafka_2.11-1.1.0/config
+sed -i '123d' server.properties
+sed -i "123a zookeeper.connect=localhost:$PAI_CONTAINER_HOST_zookeeper_PORT_LIST" server.properties
+sed -i "31a listeners=PLAINTEXT://localhost:$PAI_CONTAINER_HOST_kafka_PORT_LIST" server.properties
+cd /root/kafka_2.11-1.1.0
+service="zookeeper"
+while ! ps -ef | grep $service | egrep -v grep >/dev/null
+do
+    sleep 5s
+done
+./bin/kafka-server-start.sh config/server.properties 1>/dev/null 2>&1 &
+pid=$!
+sleep 30s
```

Comment:
Why sleep for 30s?

Reply:
To make sure that kafka server has been started.